test/extended/testdata/gssapi/fedora/base/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Clone from the Fedora 23 image
+FROM fedora:23
+
+ARG REALM
+ARG HOST
+
+ENV REALM ${REALM}
+ENV HOST ${HOST}
+
+ENV CLIENT CLIENT_HAS_LIBS
+
+ENV OS_ROOT /run/os
+ENV KUBECONFIG ${OS_ROOT}/config
+ENV PATH ${OS_ROOT}/bin:${PATH}
+
+WORKDIR ${OS_ROOT}
+ADD gssapi-tests.sh gssapi-tests.sh
+ADD test-wrapper.sh test-wrapper.sh
+ADD config config
+ADD hack hack
+ADD oc bin/oc
+
+RUN dnf install -y findutils bc
```

Comment:
every RUN statement makes a separate layer... better to combine them if possible
```
RUN dnf install -y findutils bc && \
    sed -i.bak1 's#KEYRING:persistent:#DIR:/tmp/krb5cc_#' /etc/krb5.conf && \
    chmod +x gssapi-tests.sh test-wrapper.sh
```